### ISSUE TITLE
aiohttp: 'Session is closed' on admin_create_user

### ISSUE THREAD

**Issue by ba1dr** [errietta/mandate#5]
```python
cog = Cognito(
    COGNITO['pool_id'],
    COGNITO['app_client_id'],
    user_pool_region=COGNITO['region'],
    username=adminuser['email'],
    loop=self.loop,
)
result = await cog.admin_authenticate(adminuser['password'])
temp_passw = "Test123Test123"
cog._client = None  # WORKAROUND!!
result = await cog.admin_create_user(user['email'], temporary_password=temp_passw,
                                     attr_map=None, email=user['email'], )
```

If I do not do a workaround in the example above I get an exception:

```
    result = await cog.admin_confirm_sign_up(user['email'])
  File "/opt/venv/lib/python3.6/site-packages/mandate/__init__.py", line 219, in admin_confirm_sign_up
    Username=username,
  File "/opt/venv/lib/python3.6/site-packages/aioboto3/aiobotocore/client.py", line 92, in _make_api_call
    operation_model, request_dict)
  File "/opt/venv/lib/python3.6/site-packages/aioboto3/aiobotocore/endpoint.py", line 231, in _send_request
    exception)):
  File "/opt/venv/lib/python3.6/site-packages/aioboto3/aiobotocore/endpoint.py", line 262, in _needs_retry
    caught_exception=caught_exception, request_dict=request_dict)
  File "/opt/venv/lib/python3.6/site-packages/botocore/hooks.py", line 356, in emit
    return self._emitter.emit(aliased_event_name, **kwargs)
  File "/opt/venv/lib/python3.6/site-packages/botocore/hooks.py", line 228, in emit
    return self._emit(event_name, kwargs)
  File "/opt/venv/lib/python3.6/site-packages/botocore/hooks.py", line 211, in _emit
    response = handler(**kwargs)
  File "/opt/venv/lib/python3.6/site-packages/botocore/retryhandler.py", line 183, in __call__
    if self._checker(attempts, response, caught_exception):
  File "/opt/venv/lib/python3.6/site-packages/botocore/retryhandler.py", line 251, in __call__
    caught_exception)
  File "/opt/venv/lib/python3.6/site-packages/botocore/retryhandler.py", line 269, in _should_retry
    return self._checker(attempt_number, response, caught_exception)
  File "/opt/venv/lib/python3.6/site-packages/botocore/retryhandler.py", line 317, in __call__
    caught_exception)
  File "/opt/venv/lib/python3.6/site-packages/botocore/retryhandler.py", line 223, in __call__
    attempt_number, caught_exception)
  File "/opt/venv/lib/python3.6/site-packages/botocore/retryhandler.py", line 359, in _check_caught_exception
    raise caught_exception
  File "/opt/venv/lib/python3.6/site-packages/aioboto3/aiobotocore/endpoint.py", line 305, in _get_response
    stream=streaming)
  File "/opt/venv/lib/python3.6/site-packages/aioboto3/aiobotocore/endpoint.py", line 214, in _request
    verify_ssl=verify)
  File "/opt/venv/lib/python3.6/site-packages/aiohttp/client.py", line 350, in _request
    raise RuntimeError('Session is closed')
```
Python 3.6, libraries:
```
aioboto3==6.3.0
aiobotocore==0.10.2
aiofiles==0.4.0
aiohttp==3.5.4
```

**Comment by errietta** [comment]
@ba1dr Thanks for the bug report! Will look later

**Comment by errietta** [comment]
@ba1dr  thanks a lot- this has been fixed in 0.2.5, if you can let me know if it fixes it for you :)

https://pypi.org/project/mandate/0.2.5
